assets/app/views/directives/logs/_log-viewer.html

```diff
@@ -3,6 +3,18 @@
     <span>Current log from {{start}}&nbsp;</span>
     <span ng-if="end">to {{end}}</span>
   </div>
+
+  <div row main-axis="end" ng-if="canViewArchive">
+    <form
+      ng-if="canViewArchive"
+      action="{{kibanaAuthUrl}}"
+      method="POST" target="_blank">
```

Comment:
The idea is that Kibana and the Java console should look like the web console so they feel like one integrated application. I'm not sure we've accomplished that.

I think we should be consistent in how we handle the links to both whichever way we go.